deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpsConnection.java

```diff
@@ -202,7 +215,10 @@ public void openAmqpAsync()
 
         log.debug("Starting amqp reactor thread...");
         AmqpReactor amqpReactor = new AmqpReactor(this.reactor);
-        ReactorRunner reactorRunner = new ReactorRunner(amqpReactor);
+
+        String reactorRunnerPrefix = this.hostName + "-Cxn" + this.connectionId;
+
+        ReactorRunner reactorRunner = new ReactorRunner(amqpReactor, reactorRunnerPrefix, "AmqpOpenAsync");
```

Comment:
"AmqpOpenAsync" as a thread suffix makes it sound like this thread is only devoted to opening the connection. The thread owns the connection completely, though, from open, to send, to close.